benchmarking/driver/benchmark_driver.py

```diff
@@ -108,7 +108,7 @@ def _processDelayData(input_data):
         data[k] = {
             "values": input_data[k]["values"],
             "summary": _getStatistics(input_data[k]["values"]),
-            "type": k,
+            "type": k.split()[0],
```

Comment:
Add a new field in caffe2 to specify "type" and use the value of that field directly.